<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Mirazul Islam, PhD</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-item"><a href="index.html" class="current">Matthew&nbsp;G.&nbsp;Jones</a></div>
<div class="menu-item"><a href="Matt_Jones_CV.pdf">CV</a></div>
<div class="menu-item"><a href="publications.html">publications</a></div>
<div class="menu-category">software</div>
<div class="menu-item"><a href="https://github.com/YosefLab/Cassiopeia">cassiopeia</a></div>
<div class="menu-item"><a href="https://github.com/AmpliconSuite/CoRAL">CoRAL</a></div>
<div class="menu-item"><a href="https://github.com/YosefLab/Vision">vision</a></div>
<div class="menu-item"><a href="https://github.com/russcd/selam">selam</a></div>
<div class="menu-category">connect</div>
<div class="menu-item"><a href="https://github.com/mattjones315">github</a></div>
<div class="menu-item"><a href="https://twitter.com/mattjonesucsf">twitter</a></div>
<div class="menu-item"><a href="https://www.linkedin.com/in/matthew-jones-3a751a67">linkedin</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Mirazul Islam, PhD</h1>
</div>
<table class="imgtable"><tr><td>
<img src="headshot_cropped2.png" alt="250px" width="250px" />&nbsp;</td>
<td align="left"><h2>About Me</h2>
<p>I am an NCI K99/R00 Postdoctoral Scholar at Stanford University in <a href="https://profiles.stanford.edu/howard-chang" target=&ldquo;blank&rdquo;>Howard Chang's lab</a>. Generally, I am interested in studying how tumors evolve over time, especially under various pressures exerted by the immune system and therapies. My work typically takes advantage of high-throughput sequencing assays (e.g., scRNA-seq and spatial transcriptomics), lineage-tracing technologies, and new computational tools and inference techniques.
</p>
<p>My current research focuses on building more predictive models of tumor evolution, with a focus on how circular, oncogenic extrachromosomal DNA (ecDNA) reshape traditional rules of inheritance to accelerate tumor evolution and overcome targeted therapies. A key outcome of my research thus far is contributing to the discovery that ecDNA amplifications are often co-inherited during mitosis, leading to non-coding oncogenic amplifications and new therapeutic opportunities. Previously, I developed single-cell evolving lineage tracing technologies and computational tools.  In this, I developed <a href="https://github.com/YosefLab/Cassiopeia" target=&ldquo;blank&rdquo;>Cassiopeia</a>, a software suite for reconstructing single-cell phylogenies from continuous CRISPR/Cas9 barcoding data. We used these tools to study how aggressive cancers evolve from a single transformed cell and eventually metastasize throughout a body.
</p>
<p>In 2020, I spent the summer as a research intern at <a href="https://research.google/teams/health/" target=&ldquo;blank&rdquo;>Google Brain Genomics</a> working on deep learning methods for Electronic Health Record (EHR) data.
</p>
<p>Previously, I completed my PhD in Bioinformatics at UC San Francisco and UC Berkeley advised by <a href="https://weissman.wi.mit.edu/" target=&ldquo;blank&rdquo;>Jonathan Weissman</a> and <a href="https://yoseflab.github.io/" target=&ldquo;blank&rdquo;>Nir Yosef</a>. Before my graduate work, I studied Computer Science at UC Berkeley and worked in the <a href="https://nielsen-lab.github.io/" target=&ldquo;blank&rdquo;>Nielsen Lab</a> on developing simulators for large admixture events. And before that, I grew up in a small town outside of Pasadena, CA where I was a tri-sport athelete and musician.
</p>
</td></tr></table>
<h2>Research Interests</h2>
<ul>
<li><p>Cancer evolution
</p>
</li>
<li><p>Cancer immunology
</p>
</li>
<li><p>Lineage tracing
</p>
</li>
<li><p>Genomics technology
</p>
</li>
</ul>
<h2>Selected Publications</h2>
<ol>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2024.10.21.619529v1" target=&ldquo;blank&rdquo;>Spatiotemporal lineage tracing reveals the dynamic spatial architecture of tumor growth and metastasis.</a> <b>Matthew G. Jones</b>*, Dawei Sun*, &hellip;, Fei Chen, Jonathan S. Weissman, Nir Yosef, Dian Yang. <i>BioRxiv.</i> 2024.
</p>
</li>
<li><p><a href="https://www.nature.com/articles/s41586-024-07861-8" target=&ldquo;blank&rdquo;>Coordinated inheritance of extrachromosomal DNA species in human cancer cells.</a> King L Hung*, <b>Matthew G Jones</b>*, Ivy Tzo-Lo Wong*, EJ Curtis*, &hellip;, Paul S Mischel, Howard Y Chang. Accepted in principle, <i>Nature.</i> 2024
</p>
</li>
<li><p><a href="https://www.biorxiv.org/content/10.1101/2024.02.15.580594v1" target=&ldquo;blank&rdquo;>CoRAL accurately resolves extrachromosomal DNA genome structures with long-read sequencing.</a> Kaiyuan Zhu*, <b>Matthew G. Jones*</b>, Jens Luebeck, &hellip;, Paul S. Mischel, Howard Y. Chang, Vineet Bafna. Accepted in principle, <i>Genome Research.</i> 2024. 
</p>
</li>
<li><p><a href="https://www.sciencedirect.com/science/article/pii/S0092867422004627" target=&ldquo;blank&rdquo;>Lineage Recording Reveals the Phylodynamics, Plasticity and Paths of Tumor Evolution.</a> Dian Yang*, <b>Matthew G Jones</b>*, &hellip;, Nir Yosef, Tyler Jacks, Jonathan S Weissman. <i>Cell.</i> 2022
</p>
</li>
<li><p><a href="https://science.sciencemag.org/content/early/2021/01/21/science.abc1944/" target=&ldquo;blank&rdquo;>Single-cell lineages reveal the rates, routes, and drivers of metastasis in cancer xenografts.</a> Jeffrey J. Quinn*, <b>Matthew G Jones</b>*, Ross A Okimoto, Shigeki Nanjo, Michelle M Chan, Nir Yosef, Trever G Bivona, Jonathan S Weissman. <i>Science</i>. 2021.
</p>
</li>
<li><p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02000-8" target=&ldquo;blank&rdquo;>Inference of Single Cell Phylogenies from Lineage Tracing Data with Cassiopeia.</a> <b>Matthew G Jones</b>*, Jeffrey J Quinn*, Alex Khodaverdian*, Michelle M Chan, Jeffrey A Hussmann, Robert Wang, Chenling Xu, Jonathan S Weissman, Nir Yosef. <i>Genome Biology.</i> 2020.
</p>
</li>
</ol>
<p>Find my full publication list <a href="publications.html" target=&ldquo;blank&rdquo;>here</a>.
</p>
<h2>Beyond Science</h2>
<p>In my downtime, you can find me engrossed in a <a href="booklist.html" target=&ldquo;blank&rdquo;>book</a>, playing guitar, or listening to my records. I often am looking for an excuse to get outside or to travel.
</p>
<div id="footer">
<div id="footer-text">
Page generated 2024-11-11 11:21:18 PST, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
