<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN"
  "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
<head>
<meta name="generator" content="jemdoc, see http://jemdoc.jaboc.net/" />
<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
<link rel="stylesheet" href="jemdoc.css" type="text/css" />
<title>Publications</title>
</head>
<body>
<table summary="Table for page layout." id="tlayout">
<tr valign="top">
<td id="layout-menu">
<div class="menu-item"><a href="index.html" class="current">Matthew&nbsp;G.&nbsp;Jones</a></div>
<div class="menu-item"><a href="Matt_Jones_CV.pdf">CV</a></div>
<div class="menu-item"><a href="publications.html">publications</a></div>
<div class="menu-category">software</div>
<div class="menu-item"><a href="https://github.com/YosefLab/Cassiopeia">cassiopeia</a></div>
<div class="menu-item"><a href="https://github.com/AmpliconSuite/CoRAL">CoRAL</a></div>
<div class="menu-item"><a href="https://github.com/YosefLab/Vision">vision</a></div>
<div class="menu-item"><a href="https://github.com/russcd/selam">selam</a></div>
<div class="menu-category">connect</div>
<div class="menu-item"><a href="https://github.com/mattjones315">github</a></div>
<div class="menu-item"><a href="https://twitter.com/mattjonesucsf">twitter</a></div>
<div class="menu-item"><a href="https://www.linkedin.com/in/matthew-jones-3a751a67">linkedin</a></div>
</td>
<td id="layout-content">
<div id="toptitle">
<h1>Publications</h1>
</div>
<p>Likely my <a href="https://scholar.google.com/citations?user=0dvQ0rUAAAAJ&amp;hl=en" target=&ldquo;blank&rdquo;>Google Scholar page</a> is more up to date.
</p>
<h2>2024</h2>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.21.619529v1" target=&ldquo;blank&rdquo;>Spatiotemporal lineage tracing reveals the dynamic spatial architecture of tumor growth and metastasis.</a> <b>Matthew G. Jones</b>*, Dawei Sun*, &hellip;, Fei Chen, Jonathan S. Weissman, Nir Yosef, Dian Yang. <i>BioRxiv.</i> 2024.
</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.09.04.611262v1" target=&ldquo;blank&rdquo;>Enhancer activation from transposable elements in extrachromosomal DNA.</a> Katerina Kraft*, Sedona E. Murphy*, <b>Matthew G. Jones</b>*, &hellip;, Paul S. Mishcel, Alistair N. Boettiger, Howard Y. Chang. <i>BioRxiv.</i> 2024.
</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.10.22.619657v1.abstract" target=&ldquo;blank&rdquo;>Extrachromosomal DNA driven oncogene spatial heterogeneity and evolution in glioblastoma.</a> Imran Noorani*, Magnus Haughey*, &hellip;, <b>Matthew G. Jones</b>, &hellip;, Charles Swanton, Benjamin Werner. <i>BioRxiv.</i> 2024.
</p>
<p><a href="https://www.biorxiv.org/content/10.1101/2024.08.27.610007v1.abstract" target=&ldquo;blank&rdquo;>Tree reconstruction guarantees from CRISPR-Cas9 lineage tracing data using Neighbor-Joining.</a> Sebastian Prillo, Kevin An, Wilson Wu, Ivan Kristanto, <b>Matthew G. Jones</b>, Yun S. Song, Nir Yosef. <i>BioRxiv.</i> 2024. 
</p>
<p><a href="https://aacrjournals.org/cancerdiscovery/article/doi/10.1158/2159-8290.CD-23-1117/746896/Disparate-pathways-for-extrachromosomal-DNA" target=&ldquo;blank&rdquo;>Disparate pathways for extrachromosomal DNA biogenesis and genomic DNA repair.</a> John C. Rose, Julia A. Belk, &hellip;, <b>Matthew G. Jones</b>, &hellip;, Howard Y. Chang. <i>Cancer Discovery.</i> 2024.
</p>
<p><a href="https://www.nature.com/articles/s41586-024-07861-8" target=&ldquo;blank&rdquo;>Coordinated inheritance of extrachromosomal DNA species in human cancer cells.</a> King L Hung*, <b>Matthew G Jones</b>*, Ivy Tzo-Lo Wong*, EJ Curtis*, &hellip;, Paul S Mischel, Howard Y Chang. <i>Nature.</i> 2024
</p>
<p><a href="https://genome.cshlp.org/content/34/9/1344" target=&ldquo;blank&rdquo;>CoRAL accurately resolves extrachromosomal DNA genome structures with long-read sequencing.</a> Kaiyuan Zhu*, <b>Matthew G. Jones*</b>, Jens Luebeck, &hellip;, Paul S. Mischel, Howard Y. Chang, Vineet Bafna. <i>Genome Research.</i> 2024. 
</p>
<p><a href="https://www.nature.com/articles/s43018-024-00756-7" target=&ldquo;blank&rdquo;>PERCEPTION predicts patient response and resistance to treatment using single-cell transcriptomics of their tumors.</a> Sanju Sinha, &hellip;, <b>Matthew G. Jones</b>, &hellip;, Eytan Ruppin. <i>Nature Cancer.</i> 2024.
</p>
<p><a href="https://www.nature.com/articles/s41586-024-07066-z" target=&ldquo;blank&rdquo;>Deciphering cell states and genealogies of human hematopoiesis.</a> Chen Weng, Fulong Yu, Dian Yang, Michael Poeschla, L Alexander Liggett, <b>Matthew G Jones</b>, &hellip;, Vijay G Sankaran. <i>Nature.</i> 2024.
</p>
<h2>2023</h2>
<p><a href="https://www.nature.com/articles/s41576-023-00586-w" target=&ldquo;blank&rdquo;>Best practices for single-cell analysis across modalities.</a> Lukas Heumos*, Anna C. Schaar*, &hellip;, <b>Single-cell Best Practices Consortium</b>, Fabian J. Theis. <i>Nature Reviews Genetics.</i> 2023. 
</p>
<p><a href="https://www.annualreviews.org/doi/abs/10.1146/annurev-cancerbio-061421-123301" target=&ldquo;blank&rdquo;>New Tools for Lineage Tracing in Cancer In Vivo.</a> <b>Matthew G Jones</b>*, Dian Yang*, Jonathan Weissman. <i>Annual Reviews in Cancer.</i> 2023.
</p>
<p><a href="https://www.sc-best-practices.org/trajectories/lineage_tracing.html" target=&ldquo;blank&rdquo;>Lineage Tracing.</a> <b>Matthew G. Jones</b>*, Zoe Piran*. <i><a href="https://www.sc-best-practices.org/preamble.html" target=&ldquo;blank&rdquo;>Single-cell best practices</a></i>. 2023.
</p>
<h2>2022</h2>
<p><a href="https://www.sciencedirect.com/science/article/pii/S0092867422004627" target=&ldquo;blank&rdquo;>Lineage Recording Reveals the Phylodynamics, Plasticity and Paths of Tumor Evolution.</a> Dian Yang*, <b>Matthew G Jones</b>*, &hellip;, Nir Yosef, Tyler Jacks, Jonathan S Weissman. <i>Cell.</i> 2022
</p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S2667237522000583" target=&ldquo;blank&rdquo;>PhyloVision: Interactive Software for Integrated Analysis of Single-Cell Transcriptomic and Phylogenetic Data.</a> <b>Matthew G Jones</b>*, Yanay Rosen*, Nir Yosef. <i>Cell Reports Methods.</i> 2022
</p>
<h2>2021</h2>
<p><a href="https://www.biorxiv.org/content/10.1101/2021.05.28.446021v1" target=&ldquo;blank&rdquo;>Reconstructing unobserved cellular states from paired single-cell lineage tracing and transcriptomics data.</a> Khalil Ouardini, Romain Lopez, <b>Matthew G Jones</b>, Sebastian Prillo, Richard Zhang, Michael I. Jordan, Nir Yosef. ICML Workshop in Computational Biology. 2021.
</p>
<p><a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(21)00194-0" target=&ldquo;blank&rdquo;>Benchmarked approaches for cell lineage reconstructions of in vitro dividing cells and in silico models of Caenorhabditis elegans and Mus musculus developmental trees.</a> Wuming Gong*, Alejandro Granados*, Jingyuan Hu*, <b>Matthew G. Jones</b>*, Ofir Raz*, Irepan Salvador-Martinez*, Hanrui Zhang*, &hellip;, Pablo Meyer. <i>Cell Systems</i>. 2021.
</p>
<p><a href="https://science.sciencemag.org/content/early/2021/01/21/science.abc1944/" target=&ldquo;blank&rdquo;>Single-cell lineages reveal the rates, routes, and drivers of metastasis in cancer xenografts.</a> Jeffrey J. Quinn*, <b>Matthew G Jones</b>*, Ross A Okimoto, Shigeki Nanjo, Michelle M Chan, Nir Yosef, Trever G Bivona, Jonathan S Weissman. <i>Science</i>. 2021.
</p>
<h2>2020</h2>
<p><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02000-8" target=&ldquo;blank&rdquo;>Inference of Single Cell Phylogenies from Lineage Tracing Data with Cassiopeia.</a> <b>Matthew G Jones</b>*, Jeffrey J Quinn*, Alex Khodaverdian*, Michelle M Chan, Jeffrey A Hussmann, Robert Wang, Chenling Xu, Jonathan S Weissman, Nir Yosef. <i>Genome Biology.</i> 2020.
</p>
<p><a href="https://pubs.acs.org/doi/10.1021/acschembio.0c00339" target=&ldquo;blank&rdquo;>Robset Sequence Determinants of \(\alpha\)-Synuclein Toxicity in Yeast Implicate Membrane Binding.</a> Robert W. Newberry, &hellip;, <b>Matthew G Jones</b>, &hellip;, Willaim F. Degrado, Martin Kampmann. <i>ACS Chem Biol.</i> 2020.
</p>
<h2>2019</h2>
<p><a href="https://www.nature.com/articles/s41467-019-12235-0" target=&ldquo;blank&rdquo;>Functional interpretation of single cell similarity maps.</a> David DeTomaso*, <b>Matthew G Jones</b>*, Meena Subramaniam, Tal Ashuach, Chun J. Ye, Nir Yosef. <i>Nat. Comm.</i> 2019.
</p>
<p><a href="https://www.nature.com/articles/s41586-019-1184-5" target=&ldquo;blank&rdquo;>Molecular recording of mammalian embryogenesis.</a> Michelle M Chan, Zachary D Smith, Sefanie Grosswendt, Helene Kretzmer, Thomas M Norman, Britt Adamson, Marco Jost, Jeffrey J Quinn, Dian Yang, <b>Matthew G Jones</b>, Alex Khodaverdian, Nir Yosef, Alexander Messiner, Jonathan S Weissman. <i>Nature.</i> 2019.
</p>
<h2>Before 2019</h2>
<p><a href="https://academic.oup.com/bioinformatics/article-abstract/32/19/3035/2196613" target=&ldquo;blank&rdquo;>SELAM: simulation of epistasis and local adapation during admixture with mate choice.</a> Russell Corbett-Detig, <b>Matt Jones</b>. <i>Bioinformatics</i>. 2016.
</p>
<h2>PhD Thesis</h2>
<p><a href="https://www.proquest.com/openview/06a1f8348f6d71b083d5441a76ad0d6d/1?pq-origsite=gscholar&amp;cbl=18750&amp;diss=y" target=&ldquo;blank&rdquo;>Following Tumor Progression Step-By-Step with CRISPR/Cas9-Based Single-Cell Lineage Tracing Technologies and Improved Computational Methods</a>. <b>Matthew G. Jones</b>. 2022.
</p>
<div id="footer">
<div id="footer-text">
Page generated 2024-11-11 11:21:19 PST, by <a href="https://github.com/wsshin/jemdoc_mathjax" target="blank">jemdoc+MathJax</a>.
</div>
</div>
</td>
</tr>
</table>
</body>
</html>
